arangod/Auth/UserManager.cpp

```diff
@@ -606,6 +606,7 @@ Result auth::UserManager::accessUser(std::string const& user,
   if (it != _userCache.end()) {
     return func(it->second);
   }
+  TRI_ASSERT(user != "root");
```

Comment:
I think we should remove this assertion. I am not 100% sure if there is a window of time in which the root user is not _yet_ available.